code-challenges.test.js

```diff
@@ -51,6 +125,29 @@ var arrayOfWords = ["Apple", "Banana", "Plum", "Cherry", "Kiwi", "Peach"]
 
 // --------------------3) Create a function that takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is exactly one pair and one three of a kind.
 
+//I was able to get the key values but I was unable to get an if statment to extract those key value pairs for the times the number came up. 
+// I had done other changes but this was where I was able to still get the count. 
+
+
+//setup function that takes in an array as the argument
+const fullHouse = (array) => {
+    
+//check the value of each index
+ var currentHand = {}
+
+
+for (var i = 0; i< array.length; i++){
```

Comment:
why the for-loop  😿 . Try the .forEach() haha  But this logic for holding data about the count for each suit of card is spot-on.